build/azDevOps/azure/azure-pipelines-javaspring-k8s.yml

```diff
@@ -170,7 +170,7 @@ variables:
   - name: java_project_type
     value: "Java App"
   - name: functional_test_project_type
-    value: "Functional API Tests"
+    value: "Functional API Tests2"
```

Comment:
Is it supposed to say Tests2?

Reply:
I added that to check the cli paorsin, but forget to revert it back.